### PR TITLE
Include actual blockchain height in block response [ECR-3600]

### PR DESCRIPTION
**Pull request by SergeiMal**
See: https://jira.bf.local/browse/ECR-3600

### CODE REVIEW

**Comment by aleksuss** [comment]
@SergeiMal update changelog please.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum
 
+- `explorer/v1/block` endpoint with requested a height exceeding the current
+  blockchain height, responds with error message includes the current blockchain
+  height.. (#1498)
```
Comment (aleksuss) [inline CHANGELOG.md]:
```suggestion
  height. (#1498)
```

File: testkit/tests/counter/main.rs [hunk 2]
```diff
@@ -526,6 +526,28 @@ fn test_explorer_blocks_basic() {
     );
 }
 
+#[test]
+fn test_explorer_block_base() {
```
Comment (popzxc) [inline testkit/tests/counter/main.rs]: IMHO the name of the test doesn't explain what does it check (however, that applies for other tests in that module too...).

Maybe something like `test_explorer_api_block_request`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum
 
+- `explorer/v1/block` endpoint with requested a height exceeding the current
```
[the remark this answers is not in the document]
Comment (aleksuss) [reply CHANGELOG.md]: I think yes if a light client handle this message e.g.

[the remark this answers is not in the document]
Comment (popzxc) [reply CHANGELOG.md]: In that sense any change can be named breaking: e.g. some client code can parse log output and adding a new log entry will break it.
In my opinion, breaking changes are changes that make low-level API different (renamed methods, different returned HTTP status codes, etc). String representation of the error is not appropriate thing to parse and build logic on, for that purposes we have other explorer api methods.

**Pull request merged**